Alpha_shapes_2/doc/Alpha_shapes_2/Alpha_shapes_2.txt

```diff
@@ -43,6 +43,10 @@ it's way too large. So we will never spoon up ice-cream lying in the
 inside of the convex hull of \f$ S\f$, and hence the \f$ \alpha\f$-shape for
 \f$ \alpha \rightarrow \infty\f$ is the convex hull of \f$ S\f$.
 
+\cgal offers 2D and 3D alpha shapes. The GUDHI library offers a
+<a href="http://gudhi.gforge.inria.fr/doc/latest/group__alpha__complex.html"> dD Alpha complex</a>.
```

Comment:
Hi Vincent, I just discussed the subject in-real-life with @cjamin. I did not know that the code of the `Alpha_complex` from GUDHI was beyond the simple computation of the dD alpha shape. Clément told me that there was code inside it to compute persistence on the alpha shape. And that of course is the subject of GUDHI, not pure geometry computation. I am imagine that extracting a pure-geometrical dD alpha shape code from GUDHI is quite difficult.